html-aam/index.html

```diff
@@ -16400,6 +16427,19 @@ <h3>Accessible Description Computation</h3>
                 <a data-cite="accname-1.2/#mapping_additional_nd_te">text equivalent computation</a> of the subtree of the first `caption` element if it was not used as the
                 <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>.
               </li>
+              <li>
+                an `optgroup` element which has a visible label rendered by its `label` attribute, by its <a href="https://dom.spec.whatwg.org/#concept-tree-child">child</a> `legend` element, or both:
+                  <ol>
+                    <li>
+                      use the value of the `label` attribute if it was not used as the 
+                      <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>
+                    </li>
+                    <li>
+                      or use the <a data-cite="accname-1.2/#mapping_additional_nd_te">text equivalent computation</a> of the subtree of the 
+                      first `legend` element if it was not used as the <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>.
```

Comment:
We should expose DESCRIBED_BY and DESCRIPTION_FOR relations if we do this. I think we have the same issue for table captions when used as description, but I guess we should address that separately.